tests/advection_solver_fail/screen-output

```diff
@@ -60,3 +60,9 @@ Additional information:
     
 
 Aborting!
+MPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD
```

Comment:
Would it make sense to filter these out in the default script? It certainly won't work with mpich if we leave it in...